test/WireMock.Net.Tests/WireMockServerTests.cs

```diff
@@ -196,6 +196,72 @@ public async Task WireMockServer_When_HttpClientWithWebProxyCallsHttp_Should_Wor
     }
 #endif
 
+[Fact]
+public async Task WireMockServer_WithUrl0000_Should_Listen_On_All_IPs()
+{
+    // Arrange
+
+    List<string> IPv4 = new List<string>();
+    List<string> IPv6 = new List<string>();
+
+    foreach (NetworkInterface netInterface in NetworkInterface.GetAllNetworkInterfaces())
+    {
+        if (netInterface.OperationalStatus == OperationalStatus.Up)
+        {
+
+            IPInterfaceProperties ipProps = netInterface.GetIPProperties();
+
+            foreach (UnicastIPAddressInformation addr in ipProps.UnicastAddresses)
+            {
+                Console.WriteLine(" " + addr.Address.ToString());
+
+                if (addr.Address.AddressFamily == System.Net.Sockets.AddressFamily.InterNetwork)
+                {
+                    IPv4.Add(addr.Address.ToString());
+                }
+                else if (addr.Address.AddressFamily == System.Net.Sockets.AddressFamily.InterNetworkV6)
+                {
+                    IPv6.Add($"[{addr.Address.ToString()}]");
+                }
+            }
+        }
+    }
+
+    // Asser.
```

Comment:
`// Assert`

Reply:
This was copied from another test, so I fixed the others, too-